packages/lens/codegen.yml

```
@@ -1,4 +1,4 @@
-schema: https://api-mumbai.lens.dev
+schema: https://staging-api-social-mumbai.lens.crtlkey.com
```

```suggestion
schema: https://api-mumbai.lens.dev
```

Reverting because it's deployed in prod testnet